nextjs-task-system/components/ModalTask.tsx

```
@@ -0,0 +1,256 @@
+    (async () => {
+      if (role === Role.admin) {
+        const resp = await getAllUsers(`page=1&limit=1000`);
+        if (resp && Array.isArray(resp.data)){
+          setAllUsers(resp.data);
+        }
+       
+        if (Array.isArray(watch("assignedTo"))) {
+          setUser(
+            watch("assignedTo").reduce(
+              (a: UserState, v: User) => ({
+                ...a,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)